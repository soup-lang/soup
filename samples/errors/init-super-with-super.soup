class Base
    ctor(x, y)
        this._x: x
        this._y: y

    func x()
        return this._x

    func y()
        return this._y

class Child: Base
    ctor(x) super(x, super.y())
        0
