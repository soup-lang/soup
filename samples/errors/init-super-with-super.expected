samples/errors/init-super-with-super.soup:13
    `super` not in member function or constructor
